### PR TITLE
Allow whitelisting of specific packages

### LINKED ISSUE #25: Whitelisting specific gems
Is there a way to whitelist specific gems?

For example, you might not want to whitelist all of GPLv2, but you might also know that it would be OK for the way you use a particular gem.

If that's not in there, would you accept a pull request that adds it?

### PR DESCRIPTION
**Pull request by codeodor**
Fixes #25

### CODE REVIEW

**Comment by jasonrclark** [comment]
Looks good. Thanks @codeodor!

File: spec/papers_spec.rb [hunk 1]
```diff
@@ -121,6 +121,27 @@
     ])
   end
 
+  it 'is OK with whitelisting specific gems', focus: true do
```
Comment (codeodor) [inline spec/papers_spec.rb]: @jasonrclark  oops, can you remove the `focus` here? Looks like I forgot to do that.

Comment (jasonrclark) [inline spec/papers_spec.rb]: Whoops, can do!

**Comment by jasonrclark** [comment]
Should be available now on [Rubygems as 2.4.2 :tada:](https://rubygems.org/gems/papers/versions/2.4.2)

**Pull request merged**